webhooks/assign-class-label/externalsecret.yaml

```
@@ -0,0 +1,17 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: webhook-cert
+  namespace: rhods-notebooks
+spec:
+  secretStoreRef:
+    name: nerc-secret-store
+    kind: SecretStore
+  target:
+    name: webhook-cert
+  data:
+  - secretKey: webhook-cert
+    remoteRef:
+      key: nerc/nerc-ocp-prod/rhods-notebooks/webhook-cert
+      property: webhook-cert
```

While the secret path here is correct, there is no property `webhook-cert` in the vault secret. There are two properties (`webhook.crt` and `webhook.key`) and you probably want both. I think the way to do that is:

```suggestion
data:
- secretKey: webhook-cert
  dataFrom:
    - extract:
        key: nerc/nerc-ocp-prod/rhods-notebooks/webhook-cert
```